app/views/content/cookies.html.erb

```diff
@@ -122,17 +122,31 @@
           24 hours
         </td>
       </tr>
-      <tr>
-        <td class="govuk-table__cell">
-          _gat_gtag_<%= @cookie_ga_code %>
-        </td>
-        <td class="govuk-table__cell">
-          Limits requests
-        </td>
-        <td class="govuk-table__cell">
-          10 minutes
-        </td>
-      </tr>
+      <% if request.path == '/provider/cookies' %>
+        <tr>
+          <td class="govuk-table__cell">
+            _gat_<%= ENV.fetch('GOOGLE_ANALYTICS_MANAGE', '') %>
```

Comment:
I haven't been able to replicate this cookie so don't know what it really looks like, however could we use `@cookie_ga_code` as we still have access to that?

Reply:
@cwrw  Yes sorry - we removed this then re-added it, amended here https://github.com/DFE-Digital/apply-for-teacher-training/pull/7021/commits/c0f59200dcea9b077adc90486a52e2f81df65ae3